### PR TITLE
readme update, Login basics. 

### PR DESCRIPTION
**Pull request by digitalSalt**
We missed that this is required as per Bobby for the project proposal

Gonna start adhering to the process of not merging anything to master unless it's passed by everyone, if possible

Lumped in Login basics as in bf6a22e5c1afa397c56172b35eb911ca9662a855 into this review.

### CODE REVIEW

**Comment by kevinzhangftw** [comment]
Looks good to me!

**Comment by digitalSalt** [comment]
thanks @kevinzhangftw

**Comment by digitalSalt** [comment]
Just lumped in the login changes with this pull req. 
@veriest in particular please take a look at the session management!

File: app/controllers/dummy_controller.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class DummyController < ApplicationController
```
Comment (digitalSalt) [inline app/controllers/dummy_controller.rb]: to get rid of later.

File: app/controllers/sessions_controller.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
+class SessionsController < ApplicationController
+	  skip_before_action :ensure_login, only: [:new, :create]
+
+	  def new
+	  	# will invoke new.html.erb
+	  end
+
+	  def create
```
Comment (kevinzhangftw) [inline app/controllers/sessions_controller.rb]: check that params are present! first

**Pull request merged**